Tetris/src/tetris/Tetromino.java

```diff
@@ -2,8 +2,83 @@
 
 /**
  * Template for tetromino characteristics and creation.
+ * 
+ * Contains the components necessary for a tetromino to exist within the game
+ * Minimal logic to implement selection of tetromino shape upon creation
  *
  */
 public class Tetromino {
-
+	
+	private boolean[][] shape; //this may be implemented in a different way
+						// but it must match the return type of getShape()
+	
+	private int x_location;
+	private int y_location;
+	
+	/**
+	 * Constructor for tetromino
+	 * Grabs shape character and initializes shape array based on selection
+	 * @param shape
+	 * char shape may be 'i', 'o', 's', 'z', j', or 'l'
+	 */
+	public Tetromino(char shape){
+		//if shape == 'l' shape ==  oooo
+		
+		//else if shape == 'o' shape == oo
+		//                              oo
+		
+		//etc
+	}
+	
+	/**
+	 * Retriever for shape data
+	 * Used by system and display
+	 * @return shape
+	 */
+	public boolean[][] getShape(){
```

Comment:
I think it should return a grid of the tetromino's current shape and orientation, so that the system can integrate it into the total grid.